File: Back/src/FastEventos.Domain/Evento.cs
﻿using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;
namespace FastEventos.Domain
{
    public class Evento
    {
        [Key]
        public int Id { get; set; }
        public string Local { get; set; }
        public string DataEvento { get; set; }
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public string ImagemURL { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial> RedesSociais { get; set; }
        public IEnumerable<PalestranteEvento> PalestranteEventos { get; set; }
        
    }
}

File: Front/FastEventos-App/src/app/models/Evento.ts
import { Lote } from "./Lote";
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial";

export interface Evento {
 id: number;
 local: string;
 dataEvento?: string;
 tema: string;
 qtdPessoas: number;
 telefone: string;
 email: string;
 imagemURL: string;
 lotes: Lote[];
 redesSociais: RedeSocial[];
 palestranteEventos: Palestrante[];
}
